src/jit/utils.cpp

```diff
@@ -353,7 +353,7 @@ void                dspRegMask(regMaskTP regMask, size_t minSiz)
 #endif // _TARGET_*
             }
             // We've already printed a register. Is this the end of a range?
-            else if ((regNum == REG_INT_LAST)
+            else if (regNum == REG_INT_LAST
```

Comment:
Yes, I think that's the right answer.  My preference would be to leave the code as-is, but it seems that duplicating statement is the only way to both the coding conventions and Clang happy :-)